mne/time_frequency/tests/test_multitaper.py

```diff
@@ -3,7 +3,7 @@
 from numpy.testing import assert_array_almost_equal
 from distutils.version import LooseVersion
 
-from mne.time_frequency import dpss_windows, multitaper_psd
+from mne.time_frequency.multitaper import dpss_windows, _psd_multitaper
```

Comment:
Sounds mostly useful for testing. Feel free to open a follow-up PR adding something to `mne/tests/common.py` and refactoring code to use it. I think there are a t least a few places we do this type of random object creation.